package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "telemetry-events",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
This pull request added functionality of cloning Maps, Sets, etc in a backwards-compatible manner.